packages/e2e/support/hooks.ts

```diff
@@ -1,13 +1,18 @@
-import { After, Before, BeforeAll, Status } from 'cucumber';
+import { After, Before, BeforeAll, Status, AfterAll } from 'cucumber';
 import { Builder } from 'selenium-webdriver';
 import { IMPLICIT_TIMEOUT, PAGE_LOAD_TIMEOUT, SCRIPT_TIMEOUT } from './const';
 import { envConfig } from '../env-keys';
 import { getBrowserCapabilities } from './browser-capabilities';
 import { TestRunContext } from './test-run-context';
+import { createTestUser, destroyTestUser, TestUser } from '../utils/test-user';
+
+let testUser: TestUser;
```

Comment:
I mentioned this before, but you could bring this in `BeforeAll`.

Create a new method on TestRunContext for storing `testUser` and simply consume `testUser` data from `TestRunContext` anywhere in your code.
`testUser` is bound to TestRun not test itself, so no point in storing it in `world` I think.